build/jobs/provision-deploy.yml

```diff
@@ -77,7 +77,7 @@ jobs:
             numberOfInstances = 2
             serviceName = $webAppName
             keyVaultName = "${{ parameters.keyVaultName }}".ToLower()
-            securityAuthenticationAuthority = "https://login.microsoftonline.com/$(tenant-id)"
+            securityAuthenticationAuthority = "https://sts.windows.net/$(tenant-id-guid)"
```

Comment:
Our test env has been using an invalid authority for ... not sure how long. I reuse the authority in the OSS service to check the token so I had to fix the authority.